client/new-stack-editor/lib/views/localstorageadapter.coffee

```diff
@@ -0,0 +1,12 @@
+kd = require 'kd'
+
+module.exports = class LocalStorageAdapter
+
+  constructor: ->
+    @ls = kd.singletons.localStorageController.storage 'StackEditorLayout'
```

Comment:
if you look at it, these adapters are not part of `flex-split` it's implementation of `Koding` so it's part of stack-editor, I'll shape up constants only for `flex-split`